extensions/php/languages/php/runnables.scm

```diff
@@ -0,0 +1,31 @@
+(
+    ; Class that follow the naming convention of PHPUnit test classes
+    ; and that doesn't have the abstract modifier
+    ; and have a method that follow the naming convention of PHPUnit test methods
+    ; and the method is public
+    (class_declaration
+        modifier: (_)? @modifier
+        (#not-eq? @modifier "abstract")
+        name: (_) @name
+        (#match? @name ".*Test")
+        body: (declaration_list
+            (method_declaration
+                (visibility_modifier)? @visibility
+                (#eq? @visibility "public")
+                name: (_) @run
+                (#match? @run "test.*")
+            )
+        )
+    )
+) @php-test
```

Comment:
It would be great to also support tests that are annotated with `@test` phpdoc comments: https://docs.phpunit.de/en/11.1/annotations.html#test

This can be added with a query like this (note `(#match? @_comment ".* @test .*")`)
```lisp
; Methods withs @test annotations
(class_declaration
    name: (_) @_class
	body: (declaration_list
        ((comment) @_comment
         .
         (method_declaration
            name: (_) @run)))

    (#match? @_class ".*Test$")
    (#match? @_comment ".* @test .*")
) @php-test
```

Reply:
I thought they dropped support for docblock annotations. I will update the PR to support them!